.env

```diff
@@ -4,6 +4,7 @@ VITE_SCORES_URL=https://score.snapshot.org
 VITE_ENVELOP_URL=https://core.envelop.fyi
 VITE_SIDEKICK_URL=https://sh5.co
 VITE_BROVIDER_URL=https://rpc.snapshot.org
+VOTE_STAMP_URL=https://stamp.fyi
```

Comment:
```suggestion
VITE_STAMP_URL=https://stamp.fyi
```

Reply:
Should we also changes the other occurrences of stamp url? Like for avatar `https://cdn.stamp.fyi/`

Reply:
> Should we also changes the other occurrences of stamp url? Like for avatar `https://cdn.stamp.fyi/`

We could, but it's not the same url